HOWTO

```diff
@@ -3881,13 +3881,17 @@ field was introduced or whether it's specific to some terse version):
 
         <=1, 2, 4, 8, 16, 32, >=64
 
+    I/O latencies nanoseconds::
+
+        <=2, 3, 4, 6, 8, 10, 15, 20, 35, 50, 75, 100, 175, 250, 375, 500, 625, 750, 875, 1000
+
```

Comment:
You would have to fix up the man page (fio.1) too...

Reply:
Just fixed the man page also.

Regards,
Ikegami